sandbox.py

```diff
@@ -100,7 +100,9 @@ def api(request):
 
 
 if __name__ == '__main__':
-    phpver = "php7.0"
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--phpbin", help="PHP binary, ex: php7.0", required=True)
```

Comment:
how about giving it a default value instead of making it a required flag?
I think you also want to pass the variable as a parameter to the sandbox so you don't break other projects who use this as a library.

Reply:
That was my first thought, but the only default value I would be able to set is "php" as "php7.0" will change over time. But if you are fine with having "php" as the default. I can make the change?

Stupid question, you mean something like this: "def sandbox(self, script, phpbin):"?

Reply:
yeah, something along those lines. Actually php7.0 might be a decent default value?

Reply:
Still not sure that I included what you wanted in regards to the sandbox parameter...